MAINT.md

```diff
@@ -0,0 +1,96 @@
+# Maintenance
```

Comment:
This is the import and light cleanups on some remaining documentation from the internal repo.